components/button/buttonSide.tsx

```diff
@@ -0,0 +1,12 @@
+
+export interface Props {
+  props: string;  
```

Comment:
I'm not seeing any usage of this prop, so probably we can remove it :smile:

Reply:
Thank you for your suggestion, I agree with you, we don't need the interface for the moment 😁